cluster/service/cluster_service_blackbox_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	testsupport "github.com/fabric8-services/fabric8-common/test"
 
 	"github.com/jinzhu/gorm"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
Not sure why your editor (VS Code?) keeps adding this unnecessary `uuid` package name :)

Reply:
yes, I use VSCode. I'll remove this pkg alias if it bothers you.

Reply:
That’s fine. Just curious why different VS Code installations do it differently. My VS Code keeps removing “uuid” but yours keeps adding it.

Reply:
I think `uuid` is redundant. Golang is also trying to remove it, also I am using `go 1.11.2`